libsplinter/src/node_registry/mod.rs

```diff
@@ -254,6 +254,12 @@ pub trait RwNodeRegistry: NodeRegistryWriter + NodeRegistryReader {
     ///  }
     ///```
     fn clone_box(&self) -> Box<dyn RwNodeRegistry>;
+
```

Comment:
Commit message says "Also fixes a typo in the `clone_box` doc string for `RwNodeRegistry`." but it doesnt look like that is actually included in this commit.

Reply:
Removed from the commit message